endpoints/local/endpoint.go

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) 2023 The KBase Project and its Contributors
+// Copyright (c) 2023 Cohere Consulting, LLC
+//
```

Comment:
This code is so last year!

Reply:
Indeed! Copyright law only requires an initial date, but I can certainly update the headers.

Reply:
Not sure that it's worth it, unless you are particularly bored in a meeting and/or are very concerned with not looking passé.